packages/database/src/migrations/20210107004358-AddSwapColorForLegendInLaoSchool-modifies-data.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+import { updateValues } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+// eslint-disable-next-line camelcase
+const Laos_Schools_Textbook_student_Ratio_Group_Id = '5f6d391a61f76a728e000011';
```

Comment:
Not that it matters but just curious why not camel-case here?

Reply:
It's related to a map overlay group code which is `Laos_Schools_Textbook_student_Ratio_Group`, but I agree changing to camelcase for code consistency.